doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -366,7 +366,16 @@ This section provides detailed lists of changes by :ref:`driver <drivers>`.
 Wi-Fi drivers
 -------------
 
-|no_changes_yet_note|
+* Added:
```

Comment:
Please remove the changelog entry from this PR, and add it as a suggestion in https://github.com/nrfconnect/sdk-nrf/pull/21339.